.github/labeler.yml

```diff
@@ -16,6 +16,14 @@ area/backends:
   - exporting/*
   - exporting/**/*
 
+area/build:
+  - build/*
```

Comment:
Please add everything under `build_external`

Reply:
Ah, yes, I forgot about that.